cmd/crossplane/core/core.go

```diff
@@ -29,13 +29,13 @@ import (
 	"github.com/crossplane/crossplane-runtime/pkg/controller"
 	"github.com/crossplane/crossplane-runtime/pkg/errors"
 	"github.com/crossplane/crossplane-runtime/pkg/feature"
+	"github.com/crossplane/crossplane-runtime/pkg/features"
```

Comment:
Hrm, I think I prefer keeping the actual feature flags defined closer to each consumer. On one hand this means every provider would need to define its own `EnableExternalSecretStores` flag, but any flag (like `EnableCompositionRevisions`) that is relevant to only one binary (in this case core Crossplane) doesn't need to define a flag in runtime only to use it in only one consumer of runtime.